packages/babel-types/src/index.js

```diff
@@ -398,6 +404,7 @@ export function inherits(child, parent) {
   }
 
   t.inheritsComments(child, parent);
+  traverse.copyCache(parent, child);
```

Comment:
Yeah, that was my motivation for asking. There's one other use of `babel-traverse`, but the other one is easy to remove by shifting some code around. https://github.com/babel/babel/compare/master...loganfsmyth:non-circular-types?expand=1